Test/Web/FunctionalTests/TestApps/WebAppFW45/BuildInfo.config

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DeploymentEvent xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns="http://schemas.microsoft.com/VisualStudio/DeploymentEvent/2013/06">
+  <ProjectName>WebAppFW45</ProjectName>
+  <Build type="MSBuild">
+    <MSBuild>
+      <BuildLabel kind="label">AutoGen_e6c9146c-713a-4202-80fc-8aadc055e73b</BuildLabel>
+      <BuildId kind="id">AutoGen_e6c9146c-713a-4202-80fc-8aadc055e73b</BuildId>
+      <BuildTimestamp kind="informative, summary">Wed, 12 Apr 2017 22:05:34 GMT</BuildTimestamp>
```

Comment:
Nothing for now. I don't want to integrate script into our func tests as I'm hoping that this will be brought back to VS 2017 and we could roll-back this change.